js/packages/phoenix-evals/examples/classification_example.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+import assert from "assert";
+import { generateClassification } from "../src/llm";
+import { openai } from "@ai-sdk/openai";
+
+const model = openai("gpt-4o-mini");
+
+const prompt = `
```

Comment:
Are we going to ship some built in prompts?

Reply:
I think were it makes sense we will yeah. I'm thinking we might want to build out an abstraction over a "metric" - I need to cross-check with other libraries and see what the right abstraction is here.